independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2817 @@
+            }
+          }, {
+            "name": "skip",
+            "in": "query",
+            "required": true,
+            "type": "integer",
+            "description": "The number of records to skip. This is used in the pagination of the data results to determine where to start taking the results from.",
+            "x-ms-summary": "Number of records to skip",
+            "default": 0,
+            "x-ms-visibility": "advanced"
+          }, {
+            "name": "top",
+            "in": "query",
+            "required": true,
+            "type": "integer",
+            "description": "The number of records to take. This is used in the pagination of the data results to determine how many results are retrieved. This value must be between 1 and 1000.",
+            "x-ms-summary": "Number of records to take",
+            "default": 1000,
+            "x-ms-visibility": "advanced"
+          }, {
+            "name": "start_date",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "startDate can be a date of the form yyyy, yyyy-MM or yyyy-MM-dd. When only a year is specified, the search period starts at the beginning of the year, when a year and month are specified, the search period starts at the beginning of the month",
+            "x-ms-summary": "Start date"
+          }, {
+            "name": "end_date",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "endDate can be a date of the form yyyy, yyyy-MM or yyyy-MM-dd. When only a year is specified, the search period ends at the end of the year, when a year and month are specified, the search period ends at the end of the month.",
+            "x-ms-summary": "End date"
+          }, {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "advanced",
+            "x-ms-summary": "Content type",
+            "description": "Content type"
+          }
+        ],
+        "description": "Gets formatted data in a flattened structure with each item corresponding to a single data point. Data is restricted to within a single measure category and can be optionally filtered by measure, reporting unit code, reporting unit type and reporting dates. Matching within each filter is by logical disjunction (OR) and matching across filters is by logical conjunction (AND). NOTE: There is currently a pagination restriction on this query that restricts requests to a maximum of 1000 results. Use skip and top to control this pagination. Sorting is not currently available.",
+        "operationId": "GetFlatFormattedData"
+      }
+    },
+    "/flat-data-extract/{measure-category-code}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}

```

Comment:
@paulculmsee Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.